security/pkg/pki/crypto_test.go

```diff
@@ -192,21 +231,21 @@ func TestParsePemEncodedKey(t *testing.T) {
 			keyType: reflect.TypeOf(&rsa.PrivateKey{}),
 		},
 		"Parse invalid RSA key": {
-			pem: `
------BEGIN RSA PRIVATE KEY-----
------END RSA PRIVATE KEY-----`,
+			pem:    keyInvalidRSA,
 			errMsg: "failed to parse the RSA private key",
 		},
 		"Parse ECDSA key": {
 			pem:     keyECDSA,
 			keyType: reflect.TypeOf(&ecdsa.PrivateKey{}),
 		},
 		"Parse invalid ECDSA key": {
-			pem: `
------BEGIN EC PARAMETERS-----
------END EC PARAMETERS-----`,
+			pem:    keyInvalidECDSA,
 			errMsg: "failed to parse the ECDSA private key",
 		},
+		"Parse PKCS8 key using RSA algorithm": {
```

Comment:
In TestParsePemEncodedKey(), both RSA key and ECDSA key have two test cases, one for valid key and one for invalid key. To be consistent with the test cases of RSA key and ECDSA key, should PKCS8 key also have two test cases (one for a valid key and one for invalid key)?

Reply:
Done.